scripts/common/build_clang.sh

```diff
@@ -14,14 +14,14 @@ cd "llvm-project-llvmorg-${ASWF_CLANG_VERSION}/llvm"
 mkdir build
 cd build
 
-if [[ $ASWF_CLANG_VERSION == 12.* || $ASWF_CLANG_VERSION == 13.* ]]; then
+if [[ $ASWF_CLANG_VERSION != 7.* && $ASWF_CLANG_VERSION != 8.* && $ASWF_CLANG_VERSION != 9.* && $ASWF_CLANG_VERSION != 10.* && $ASWF_CLANG_VERSION != 11.* ]]; then
     # Recent llvm requires python3 to build
     export PATH=/tmp/pytmp/bin:$PATH
     export LD_LIBRARY_PATH=/tmp/pytmp/lib:$LD_LIBRARY_PATH
-    ASWF_INSTALL_PREFIX=/tmp/pytmp ASWF_PYTHON_VERSION=3.9.7 ASWF_NUMPY_VERSION=1.20 /tmp/build_python.sh
+    ASWF_INSTALL_PREFIX=/tmp/pytmp ASWF_PYTHON_VERSION=3.9.11 ASWF_NUMPY_VERSION=1.20 /tmp/build_python.sh
 fi
 
-if [[ $ASWF_CLANG_VERSION == 13.* ]]; then
+if [[ $ASWF_CLANG_VERSION == 13.* || $ASWF_CLANG_VERSION == 14.* ]]; then
```

Comment:
I like what you did above, which catches CLANG 12.* and newer thus in theory keeps working with newer version, whereas this test would need to be explicitly updated when CLANG 15.* needs to be supported.

Found this [Bash script that claims to do semver comparisons](https://github.com/Ariel-Rodriguez/sh-semversion-2), would that be useful?

Reply:
Thanks! I'm hoping to get away from bash if I can, I don't feel like adding this code yet...